app_dart/bin/server.dart

```diff
@@ -193,10 +193,10 @@ Future<void> main() async {
         delegate: GetStatus(config),
       ),
 
-      /// Return supported branches.
+      /// Get supported branches for the framework repo.
       ///
-      /// Returns branches that are supported by the dashboard
-      /// for the default repo.
+      /// Get list of supported branches to run infrastructure
```

Comment:
I wasn't sure what " run infrastructure" meant. Is it correct to say "run infrastructure tasks"?

See change to this comment and if it still makes sense.

Reply:
sgtm